src/flag_attn/testing/flash.py

```diff
@@ -21,7 +22,10 @@ def attention(q, k, v, causal, sm_scale=None, upcast=False):
     ms = torch.arange(q_seq_len, device=device).unsqueeze(-1)
     ns = torch.arange(kv_seq_len, device=device)
     
-    S = torch.matmul(q, k.transpose(2, 3)) * sm_scale
+    if bias is None: 
+        S = torch.matmul(q, k.transpose(2, 3)) * sm_scale
+    else: 
+        S = torch.matmul(q, k.transpose(2, 3) + bias) * sm_scale
```

Comment:
There is something wrong here. According to the formula, 

S should be `torch.matmul(q, k.transpose(2, 3)) * sm_scale + bias` where ther is a bias.
This results in some Error like 

`RuntimeError: The size of tensor a (128) must match the size of tensor b (512) at non-singleton dimension 2`

when I run the tests.

Althought we have not built CI, It is adviced to run the test to make sure the commit could pass the tests.

Reply:
oops. yes fixed.